public/ExportButtons/libreExportButtons.js

```diff
@@ -785,6 +830,34 @@ if (!(navigator.webdriver || window.matchMedia('print').matches) && !LibreTexts?
         document.body.append(donorBoxScript);
       }
 
+      const conductorProjectID = await getBookProjectID();
+      if(conductorProjectID){
```

Comment:
We only want to show if logged in (author or higher); use `if (conductorProjectID && isPro)`